security/pkg/stsservice/tokenmanager/tokenmanager.go

```diff
@@ -92,6 +92,12 @@ func CreateTokenManager(tokenManagerType string, config Config) stsservice.Token
 	return tm
 }
 
+func (projectInfo GCPProjectInfo) GKEClusterURL() string {
```

Comment:
Where is this function used?

Reply:
Will be used in a separate PR ( I'm keeping them small so it's easier to review ).

Reply:
https://github.com/istio/istio/pull/26483/files

Reply:
Can we add this line of change in that PR instead?